lib/storages/channels_database.ts

```diff
@@ -380,7 +381,7 @@ export class PostgresChannelsDatabase extends AbstractChannelsDatabase<EnginePos
   allOpen (): Promise<PaymentChannel[]> {
     return this.engine.exec((client: any) => client.query(
       'SELECT "channelId", kind, sender, receiver, value, spent, state, "contractAddress" FROM channel ' +
-      'WHERE state < 2'
+      'WHERE state = 0'
```

Comment:
Given the new `Unidirectional` contract, only `state=0` channels should be returned.